pep-0001.txt

```diff
@@ -185,8 +185,11 @@ corrected by the editors).
 The standard PEP workflow is:
 
 * You, the PEP author, fork the `PEP repository`_, and create a file named
-  ``pep-9999.rst`` that contains your new PEP.  Use "9999" as your draft PEP
-  number.
+  ``pep-NNNN.rst`` that contains your new PEP.  ``NNNN`` should be the next
```

Comment:
```suggestion
  :file:`pep-{NNNN}.rst` that contains your new PEP.  :samp:`{NNNN}` should be the next
```

Using the `:file:`/`:samp:` role you can appropriately highlight this as a example replacement